src/patternfly/components/Splitter/splitter.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-splitter {
+  // splitter
+  --pf-c-splitter--Height: var(--pf-global--spacer--sm);
```

Comment:
I don't know that we want to use a spacer var here. If someone were to theme the global spacer vars, would we expect that change to impact the splitter size?

Reply:
great point - yeah i dont think so!